pages/1.13/security/oss/managing-users/local-users/index.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: layout.pug
+navigationTitle: Local users
+title: Local users
+excerpt: Managing local DC/OS users
+menuWeight: 20
+---
+
+<!-- The source repository for this topic is https://github.com/dcos/dcos-docs-site -->
+
+By default no local user exists in DC/OS.
+
+# Add local users with the IAM API
+
+Local users can currently only be added via the DC/OS [IAM API](/1.13/security/oss/iam-api/).
+
+The IAM API allows for managing the full local user life-cycle:
+
+- Login
+- Create
+- Update
+- Delete
```

Comment:
Somewhere in this documentation page we should say how to create a regular user account. Where do we do that? Maybe I oversee something. From the JIRA ticket:

"Document creating a user via uid/password."

Here I see login via CLI explained, but not creation.